mungegithub/Makefile

```diff
@@ -109,9 +109,11 @@ push_volume: volume
 ifneq ("", "$(and $(wildcard $(APP)/pv.yaml), $(wildcard $(APP)/pvc.yaml))")
 	# Creating persistent volume and persistent volume claim.
 	kubectl --kubeconfig=$(KUBECONFIG) create -f $(APP)/local.pv.yaml
+	# Sleeping for 1s to allow persistent volume to be created.
+	@sleep 1
 	kubectl --kubeconfig=$(KUBECONFIG) create -f $(APP)/local.pvc.yaml
-	# Sleeping for 2s to allow claim to bind to volume.
-	@sleep 2
+	# Sleeping for 3s to allow claim to bind to volume.
```

Comment:
I'm not sure how long it normally takes to bind. There was usually enough time to bind when it was set to 2 seconds, but that wasn't long enough last time I deployed. So I'm guessing 1.5 seconds as the average and changing the sleep time to 4 seconds.